### PR TITLE
sql-statements: add statement reference for ROLES

### PR DESCRIPTION
**Pull request by Joyinqin**
### What is changed, added or deleted? (Required)
update docs and some examples associated with ROLES (RBAC).

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/3198, https://github.com/pingcap/docs/pull/3208
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [x] Delete files
- [ ] Change aliases
- [x] Have version specific changes 
- [x] Might cause conflicts

### CODE REVIEW

File: sql-statements/sql-statement-set-default-role.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+SHOW TABLES IN test;
++----------------+
+| Tables_in_test |
++----------------+
+| t1             |
++----------------+
+1 row in set (0.00 sec)
+```
+
+`SET DEFAULT ROLE` 语句不会自动 `GRANT` 用户相关角色。若尝试为 `jennifer` 尚未授予的角色启用 `SET DEFAULT ROLE` 会导致以下错误：
```
Comment (TomShawn) [inline sql-statements/sql-statement-set-default-role.md:138]:
```suggestion
`SET DEFAULT ROLE` 语句不会自动将相关角色授予 (`GRANT`) 用户。若尝试为 `jennifer` 尚未被授予的角色执行 `SET DEFAULT ROLE` 会导致以下错误：
```

**Comment by TomShawn** [comment]
@bb7133 PTAL, thanks!

File: sql-statements/sql-statement-create-role.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+$ mysql -ujennifer
+Welcome to the MySQL monitor.  Commands end with ; or \g.
+Your MySQL connection id is 32
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-create-role.md:67]: This part could be removed.

File: sql-statements/sql-statement-create-role.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+Your MySQL connection id is 34
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-create-role.md:118]: ditto

File: sql-statements/sql-statement-create-role.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+Your MySQL connection id is 35
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-create-role.md:136]: ditto

File: sql-statements/sql-statement-drop-role.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+Your MySQL connection id is 37
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-drop-role.md:36]: This part could be removed.

File: sql-statements/sql-statement-drop-role.md [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+Your MySQL connection id is 32
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-drop-role.md:65]: ditto

File: sql-statements/sql-statement-drop-role.md [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+Your MySQL connection id is 34
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-drop-role.md:114]: ditto

File: sql-statements/sql-statement-drop-role.md [hunk 8]
```diff
@@ -0,0 +1,212 @@
...
+Your MySQL connection id is 35
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-drop-role.md:132]: ditto

File: sql-statements/sql-statement-drop-role.md [hunk 9]
```diff
@@ -0,0 +1,212 @@
...
+Your MySQL connection id is 41
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-drop-role.md:167]:
```suggestion
```

File: sql-statements/sql-statement-drop-role.md [hunk 10]
```diff
@@ -0,0 +1,212 @@
...
+Your MySQL connection id is 42
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-drop-role.md:187]:
```suggestion
```

File: sql-statements/sql-statement-grant-role.md [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+Your MySQL connection id is 37
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-grant-role.md:40]:
```suggestion
```

File: sql-statements/sql-statement-grant-role.md [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+Your MySQL connection id is 32
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-grant-role.md:66]:
```suggestion
```

File: sql-statements/sql-statement-grant-role.md [hunk 13]
```diff
@@ -0,0 +1,166 @@
...
+Your MySQL connection id is 34
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-grant-role.md:115]:
```suggestion
```

File: sql-statements/sql-statement-grant-role.md [hunk 14]
```diff
@@ -0,0 +1,166 @@
...
+Your MySQL connection id is 35
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-grant-role.md:133]:
```suggestion
```

File: sql-statements/sql-statement-revoke-role.md [hunk 15]
```diff
@@ -0,0 +1,211 @@
...
+Your MySQL connection id is 37
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-revoke-role.md:40]:
```suggestion
```

File: sql-statements/sql-statement-revoke-role.md [hunk 16]
```diff
@@ -0,0 +1,211 @@
...
+Your MySQL connection id is 32
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-revoke-role.md:69]:
```suggestion
```

File: sql-statements/sql-statement-revoke-role.md [hunk 17]
```diff
@@ -0,0 +1,211 @@
...
+Your MySQL connection id is 34
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-revoke-role.md:118]:
```suggestion
```

File: sql-statements/sql-statement-revoke-role.md [hunk 18]
```diff
@@ -0,0 +1,211 @@
...
+Your MySQL connection id is 35
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-revoke-role.md:136]:
```suggestion
```

File: sql-statements/sql-statement-revoke-role.md [hunk 19]
```diff
@@ -0,0 +1,211 @@
...
+Your MySQL connection id is 38
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-revoke-role.md:171]:
```suggestion
```

File: sql-statements/sql-statement-revoke-role.md [hunk 20]
```diff
@@ -0,0 +1,211 @@
...
+Your MySQL connection id is 39
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-revoke-role.md:189]:
```suggestion
```

File: sql-statements/sql-statement-set-default-role.md [hunk 21]
```diff
@@ -0,0 +1,176 @@
...
+Your MySQL connection id is 37
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-set-default-role.md:44]:
```suggestion
```

File: sql-statements/sql-statement-set-default-role.md [hunk 22]
```diff
@@ -0,0 +1,176 @@
...
+Your MySQL connection id is 32
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-set-default-role.md:70]:
```suggestion
```

File: sql-statements/sql-statement-set-default-role.md [hunk 23]
```diff
@@ -0,0 +1,176 @@
...
+Your MySQL connection id is 34
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-set-default-role.md:119]:
```suggestion
```

File: sql-statements/sql-statement-set-default-role.md [hunk 24]
```diff
@@ -0,0 +1,176 @@
...
+Your MySQL connection id is 35
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (imtbkcat) [inline sql-statements/sql-statement-set-default-role.md:137]:
```suggestion
```

File: sql-statements/sql-statement-create-role.md [hunk 25]
```diff
@@ -0,0 +1,135 @@
...
+Your MySQL connection id is 37
+Server version: 5.7.25-TiDB-v4.0.0-beta.2-728-ga9177fe84 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+
+Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
```
Comment (TomShawn) [inline sql-statements/sql-statement-create-role.md:40]:
```suggestion
```

**Comment by TomShawn** [comment]
LGTM
@imtbkcat PTAL again

File: TOC.md [hunk 26]
```diff
@@ -319,6 +323,7 @@
       - [`SHOW CREATE USER`](/sql-statements/sql-statement-show-create-user.md)
       - [`SHOW DATABASES`](/sql-statements/sql-statement-show-databases.md)
       - [`SHOW DRAINER STATUS`](/sql-statements/sql-statement-show-drainer-status.md)
+      - [`SHOW DEFAULT ROLE`](/sql-statements/sql-statement-set-default-role.md)
```
Comment (lilin90) [inline TOC.md:326]:
```suggestion
```

File: TOC.md [hunk 27]
```diff
@@ -299,6 +302,7 @@
       - [`REPLACE`](/sql-statements/sql-statement-replace.md)
       - [`RESTORE`](/sql-statements/sql-statement-restore.md)
       - [`REVOKE <privileges>`](/sql-statements/sql-statement-revoke-privileges.md)
+      - [`REVOKE <role>`](/sql-statements/sql-statement-revoke-role.md)
       - [`ROLLBACK`](/sql-statements/sql-statement-rollback.md)
       - [`SELECT`](/sql-statements/sql-statement-select.md)
```
Comment (lilin90) [inline TOC.md:307]:
```suggestion
      - [`SELECT`](/sql-statements/sql-statement-select.md)
      - [`SET DEFAULT ROLE`](/sql-statements/sql-statement-set-default-role.md)
```

**Review by lilin90 — CHANGES_REQUESTED** [review]
@Joyinqin Please fix errors. @TomShawn FYI.

File: sql-statements/sql-statement-set-role.md [hunk 28]
```diff
@@ -108,6 +108,15 @@ SELECT CURRENT_ROLE();
 1 row in set (0.000 sec)
 ```
 
+## MySQL 兼容性
+
+`SET ROLE` 语句与 MySQL 8.0 的角色功能完全兼容。如有任何其他兼容性差异，请在 GitHub 上提交 [issue](/report-issue.md)。
```
Comment (lilin90) [inline sql-statements/sql-statement-set-role.md:113]:
```suggestion
`SET ROLE` 语句与 MySQL 8.0 的角色功能完全兼容。如有任何其他兼容性差异，请在 GitHub 上提交 [issue](https://github.com/pingcap/tidb/issues/new/choose)。
```

File: sql-statements/sql-statement-set-default-role.md [hunk 29]
```diff
@@ -0,0 +1,132 @@
...
+`SET DEFAULT ROLE` 语句不会自动将相关角色授予 (`GRANT`) 用户。若尝试为 `jennifer` 尚未被授予的角色执行 `SET DEFAULT ROLE` 语句会导致以下错误：
+
+```sql
+SET DEFAULT ROLE analyticsteam TO jennifer;
+ERROR 3530 (HY000): `analyticsteam`@`%` is is not granted to jennifer@%
+```
+
+## MySQL 兼容性
+
+`SET DEFAULT ROLE` 语句与 MySQL 8.0 的角色功能完全兼容。如有任何其他兼容性差异，请在 GitHub 上提交 [issue](/report-issue.md)。
```
Comment (lilin90) [inline sql-statements/sql-statement-set-default-role.md:123]:
```suggestion
`SET DEFAULT ROLE` 语句与 MySQL 8.0 的角色功能完全兼容。如有任何其他兼容性差异，请在 GitHub 上提交 [issue](https://github.com/pingcap/tidb/issues/new/choose)。
```

File: sql-statements/sql-statement-revoke-role.md [hunk 30]
```diff
@@ -0,0 +1,145 @@
...
+| Grants for User                      |
++--------------------------------------+
+| GRANT USAGE ON *.* TO 'jennifer'@'%' |
++--------------------------------------+
+1 row in set (0.00 sec)
+```
+
+## MySQL 兼容性
+
+`REVOKE <role>` 语句与 MySQL 8.0 的角色功能完全兼容。如有任何其他兼容性差异，请在 GitHub 上提交 [issue](/report-issue.md)。
```
Comment (lilin90) [inline sql-statements/sql-statement-revoke-role.md:136]:
```suggestion
`REVOKE <role>` 语句与 MySQL 8.0 的角色功能完全兼容。如有任何其他兼容性差异，请在 GitHub 上提交 [issue](https://github.com/pingcap/tidb/issues/new/choose)。
```

File: sql-statements/sql-statement-grant-role.md [hunk 31]
```diff
@@ -0,0 +1,122 @@
...
+| Tables_in_test |
++----------------+
+| t1             |
++----------------+
+1 row in set (0.00 sec)
+```
+
+## MySQL 兼容性
+
+`GRANT <role>` 语句与 MySQL 8.0 的角色功能完全兼容。如有任何其他兼容性差异，请在 GitHub 上提交 [issue](/report-issue.md)。
```
Comment (lilin90) [inline sql-statements/sql-statement-grant-role.md:112]:
```suggestion
`GRANT <role>` 语句与 MySQL 8.0 的角色功能完全兼容。如有任何其他兼容性差异，请在 GitHub 上提交 [issue](https://github.com/pingcap/tidb/issues/new/choose)。
```

File: sql-statements/sql-statement-drop-role.md [hunk 32]
```diff
@@ -0,0 +1,146 @@
...
++--------------------------------------+
+1 row in set (0.00 sec)
+
+SET ROLE analyticsteam;
+ERROR 3530 (HY000): `analyticsteam`@`%` is is not granted to jennifer@%
+```
+
+## MySQL 兼容性
+
+`DROP ROLE` 语句与 MySQL 8.0 的角色功能完全兼容。如有任何其他兼容性差异，请在 GitHub 上提交 [issue](/report-issue.md)。
```
Comment (lilin90) [inline sql-statements/sql-statement-drop-role.md:137]:
```suggestion
`DROP ROLE` 语句与 MySQL 8.0 的角色功能完全兼容。如有任何其他兼容性差异，请在 GitHub 上提交 [issue](https://github.com/pingcap/tidb/issues/new/choose)。
```

File: sql-statements/sql-statement-create-role.md [hunk 33]
```diff
@@ -0,0 +1,145 @@
...
+| Tables_in_test |
++----------------+
+| t1             |
++----------------+
+1 row in set (0.00 sec)
+```
+
+## MySQL 兼容性
+
+`CREATE ROLE` 语句与 MySQL 8.0 的角色功能完全兼容。如有任何其他兼容性差异，请在 GitHub 上提交 [issue](/report-issue.md)。
```
[the remark this answers is not in the document]
Comment (lilin90) [reply sql-statements/sql-statement-create-role.md:115]: - @Joyinqin I see that you hasn't address @TomShawn's comment but marked it as resolved. Also for other files in this PR. Please check.
- Please use the filing issue link of the tidb repo: https://github.com/pingcap/tidb/issues/new/choose

**Review by imtbkcat — APPROVED** [review]
LGTM

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**